source/_integrations/proximity.markdown

```diff
@@ -20,23 +20,31 @@ Some examples of its use include:
 - Increase thermostat temperature as you near home
 - Decrease temperature the further away from home you travel
 
-The Proximity entity which is created has the following values:
-
-- `state`: Distance from the monitored zone (in `unit_of_measurement`)
-- `dir_of_travel`: Direction of the closest device or person to the monitored zone. Values are:
-  - `not set`
-  - `arrived`
-  - `towards`
-  - `away_from`
-  - `unknown`
-  - `stationary`
-- `unit_of_measurement`: Measurement of distance. Values are:
-  - `km`
-  - `m`
-  - `mi`
-  - `yd`
-  - `ft`
-- `nearest`: The device or person which is nearest to the zone
+## Sensors
+
+The following sensor entities will be created.
+
+### Distance
+
+For each tracked device or person, a sensor showing the distance from the monitored zone in a unit depending on your [Home Assistant Unit System](/docs/configuration/basic) selection is created.
+You can use the [Min/Max](/integrations/min_max) integration to determine the nearest and furthest distance.
+
+### Direction of travel
+
+For each tracked device or person, a sensor showing the direction of travel to or from the monitored zone is created. Possible states are:
+
+- `not_set`
+- `arrived`
+- `towards`
+- `away_from`
+- `unknown`
```

Comment:
The order here seems a bit awkward.

Did we keep `not_set`?

Reply:
you're right - we have removed `not_set` 👍